src/components/RouteWrapper.tsx

```diff
@@ -273,6 +274,13 @@ export const ROUTES: RouteProperties[] = [
         description: "Arjun can find query parameters for URL endpoints.",
         category: "Web Application Testing",
     },
+    {
+        name: "Arpscan",
```

Comment:
There seems to be some inconsistency with the way you punctuate Arpscan. In the tool itself, you have spelt it as ArpScan. The Kali Linux site spells it as Arp-Scan. Please decide on a consistent punctuation/spelling.

Reply:
Fixed it.